mil_common/utils/mil_tools/scripts/mil-preflight/menus.py

```diff
@@ -0,0 +1,152 @@
+################################################################################
+#  File name: menus.py
+#  Author: Keith Khadar
+#  Description: This file is used to store all the menus used by preflight. Version 1.0
+################################################################################
+# ----- Imports  -----#
+import tests
+
+# -----  Home Page  -----#
+title = """
+[bold green]Preflight Program - Autonomous Robot Verification[/bold green]
+"""
+info_page = """
+Welcome to the Preflight Program, a tool inspired by the preflight checklists used by pilots before flying a plane. This program is designed to verify the functionality of all software and hardware systems on your autonomous robot. It ensures that everything is in working order, allowing you to safely deploy your robot with confidence.\n
+[italic]Authors:[/italic]
+Keith Khadar
+Anthony Liao
+Joshua Thomas
+Maanas Kotha\n
```

Comment:
Please remove the authors here. We appreciate your work, and you can put it in other places, but here it takes up a lot of screen space that might be useful in a small tmux window :)

Reply:
Removed